godotopenxrlynx/README.md

```diff
@@ -0,0 +1,28 @@
+# Godot OpenXR Lynx Android loader
+
+Binaries taken from Lynx at:
+https://portal.lynx-r.com/downloads/item/1/download/latest
+
+# AndroidManifest changes
+
+Add this to your manifest to enable OpenXR and Ultraleap's hand tracking:
+
+```
+<uses-feature android:name="android.hardware.vr.headtracking" android:required="true" android:version="1"/>
+<uses-permission android:name="org.khronos.openxr.permission.OPENXR"/>
+
+<queries>
+    <intent>
+        <action android:name="org.khronos.openxr.OpenXRRuntimeService"/>
+    </intent>
+    <provider android:authorities="org.khronos.openxr.runtime_broker;org.khronos.openxr.system_runtime_broker" />
+    <package android:name="com.ultraleap.tracking.service"/>
+    <package android:name="com.ultraleap.openxr.api_layer"/>
+</queries>
+```
+
+And add this to the `application` block in your manifest to enable hand tracking:
+
+```
+<meta-data android:name="handtracking" android:value="1" />
```

Comment:
Should hand tracking be always enable since it's one of the Lynx primary input mechanism? If so, then it can be included in the plugin's manifest as mentioned above.

Reply:
There is already a setting to enable/disable the hand tracking in the export dialog in Godot but it only works for Meta's hand tracking. Ideally this should be more generic, or as seen above, add more vendor-specific options. It feels alright to leave it enabled by default for now as this will be Lynx's primary input as you said. I've added it in the AndroidManifest along with Ultraleap's vendor specific fields.